apps/matrix-io/src/matrix_io/fabricator.py

```diff
@@ -0,0 +1,388 @@
+"""Misc Fabricator functions for building yaml from real data."""
```

Comment:
Some more documentation would be helpful, what is now the process for generating this? Is this limited to generating edge/nodes files only for KG sources?

Reply:
This cli app produces the nodes & edges yaml from real data, intending to be a drop-in to a section of the fabricator's parameters.yml file used in the pipeline.